styleIndex.css

```diff
@@ -0,0 +1,27 @@
+h1{
+    font-family: 'Courier New', Courier, monospace;
+    font-size: 60px;
+    background-color: antiquewhite;
+    color: black;
+}
+h2{
+    font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+ }
+ .primera{
+     text-align: center;
+ }
+.lista{
+    text-align: center;
+}
```

Comment:
good job styling but, you can also use flexbox and grid to make it look even better.

Reply:
[OPTIONAL] Please refer to my comment regarding giving a solid reason for requesting the specific change. You can also demonstrate some code samples and give some hints. It is just the student might be not comfortable working with `flex` or `grid` or they can not understand which one to use when so giving some hints along with some code samples would benefit the student so much.